sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -3,14 +3,19 @@
     <div class="mx-5 py-4">
       <body-header title="Profile" class="m-1">
         <template #actions>
-          <v-btn
-            class="float-right"
-            @click="editProfile"
-            variant="text"
-            color="primary"
-            prepend-icon="fa:fa fa-gear"
-            >Edit
-          </v-btn>
+          <check-permission-role :role="Role.InstitutionEditProfile">
+            <template #="{ notAllowed }">
+              <v-btn
+                class="float-right"
+                @click="editProfile"
+                variant="text"
+                :color="!notAllowed ? 'primary' : 'secondary'"
```

Comment:
The check can be inverted to avoid the "!": `notAllowed ? 'secondary' : 'primary'`.
It was also a temporary solution. Maybe in a newer version of Vuetify, it will be better.

Reply:
yup